ios/RNSplashScreen.m

```diff
@@ -20,9 +22,53 @@ - (dispatch_queue_t)methodQueue{
 RCT_EXPORT_MODULE(SplashScreen)
 
 NSInteger const RNSplashScreenOverlayTag = 39293;
+NSString* RNSplashScreenOverlayName = @"splashscreenVideo";
+
++ (void)showVideo {
+  if (showingVideo || showing) return;
+
+  NSString *videoPath=[[NSBundle mainBundle] pathForResource:@"splashscreen" ofType:@"mp4"];
+  if (videoPath == nil) return;
+  showingVideo = true;
+
+  UIView *rootView = UIApplication.sharedApplication.keyWindow.subviews.lastObject;
+
+  NSURL *videoURL = [NSURL fileURLWithPath:videoPath];
+  AVPlayer *player = [AVPlayer playerWithURL:videoURL];
+  playerLayer.videoGravity = AVLayerVideoGravityResizeAspectFill;
+
+  AVPlayerLayer *playerLayer = [AVPlayerLayer playerLayerWithPlayer:player];
+  playerLayer.frame = rootView.bounds;
+  [playerLayer setName:RNSplashScreenOverlayName];
+
+  [rootView.layer addSublayer:playerLayer];
```

Comment:
Any reason to use layer instead of view?

Reply:
Attaching view directly didn't seem to work. Maybe I was doing something wrong though

Reply:
Layer is just fine, was curious.